README.md

```diff
@@ -37,7 +37,7 @@ Once you've installed and verified that these dependencies are working as expect
 -   Create a file named `.env` at the root of the repository
 -   Copy the contents of `sample.env` to your new `.env`
 -   Edit `.env` to set MONGO_URL=mongodb://localhost:27017
--   Edit `.env` to set JWT_SECRET to an _actual_ secret, such as a plain random string
+-   Edit `.env` to set JWT*SECRET to an \_actual* secret, such as a plain random string
```

Comment:
Pretty sure this isn't what you want either:

![image](https://user-images.githubusercontent.com/5133649/107238577-adb18980-6a30-11eb-8349-a02c6f54bbf2.png)

```suggestion
-   Edit `.env` to set `JWT_SECRET` to an _actual_ secret, such as a plain random string
```

Reply:
yeah i think prettier keeps messing with this; i'll put it on the ignore list